### PR TITLE
Port BN module to v8

### PR DESCRIPTION
**Pull request by Ehtaga**

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__author__ = 'amolle'
```
Comment (bwrsandman) [inline __init__.py]: It looks like this file was added by accident.

File: res_partner_attributes_add_BN/README.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Credits
+=======
+
+Contributors
+------------
+
+* Joao Alfredo Gama Batista <joao.gama@savoirfairelinux.com>
+* Marc Cassuto <marc.cassuto@savoirfairelinux.com>
+* Sandy Carter <sandy.carter@savoirfairelinux.com>
+* Agathe Mollé <agathe.molle@savoirfairelinux.com>
```
Comment (bwrsandman) [inline res_partner_attributes_add_BN/README.rst]: With this section, the AUTHORS.txt can be removed.

File: res_partner_attributes_add_BN/i18n/res_partner_attributes_add_BN.pot [hunk 3]
```diff
@@ -1,20 +1,20 @@
 # Translation of OpenERP Server.
```
Comment (bwrsandman) [inline res_partner_attributes_add_BN/i18n/res_partner_attributes_add_BN.pot]: Careful, the .pot should not have translations. They are the files that are exported from Odoo as templates.

**Comment by bwrsandman** [comment]
The column rename from `ne` to `bn` requires a simple migration script:

It goes in a subfolder with the version (odooversion.moduleversion) of the `migrations` folder with `pre-*` to be run before the module update (as opposed to `post-*`).

You will need to use and copy the openupgrade [`column_rename`](https://doc.therp.nl/openupgrade/API.html#openupgrade.rename_columns) function and call it in a [`migrate`](https://doc.therp.nl/openupgrade/API.html#openupgrade.migrate) function

``` python
# res_partner_attributes_add_BN/migrations/8.0.1.0/pre-migration.py

column_spec = {
    'res_partner': [
        ("ne", "bn"),
    ],
}


def rename_columns(cr, column_spec):
    # Copy from https://github.com/OpenUpgrade/OpenUpgrade/blob/8.0/openerp/openupgrade/openupgrade.py#L179


def migrate(cr, version):
    if version is None:
        return
    rename_columns(cr, column_spec):

```

**Comment by bwrsandman** [comment]
:+1:

File: res_partner_attributes_add_BN/README.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+=============
+
+
+Usage
+=====
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
```
Comment (max3903) [inline res_partner_attributes_add_BN/README.rst:26]: "Go to a partner form and fill in the Business identification Number, located under the email"

Comment (Ehtaga) [inline res_partner_attributes_add_BN/README.rst:26]: Done

**Comment by max3903** [comment]
:+1:

**Pull request merged**